### DISCUSSION TITLE
Image update Automation

### DISCUSSION THREAD

**Discussion by ZeroDeltaAlpha** [fluxcd/flux2#1432] [Q&A]
Hi All,

we have just started using flux V2 and encountered the following issue,
the image update automation applies an update but after doing so then falls into a status of

`│   Normal  error   14m (x86 over 17h)  image-automation-controller  object not found `

Is there anyway to see what object the automation controller is referring to such as debug levels or similar ?

Many Thanks in advance

Edit: This issue *seems* to happen when a merged branch isn't removed

**Comment by stefanprodan** [comment]
Please open an issue here https://github.com/fluxcd/image-automation-controller/issues
